Gemfile

```diff
@@ -34,7 +34,10 @@ gem 'activeresource', require: 'active_resource'
 gem 'jquery-rails'
 gem 'jquery-ui-rails'
 gem 'lograge'
-gem 'airbrake', '~> 5.2'
+
+group :staging, :production do
```

Comment:
Theres a config to turn this off, rather than doing this. It's in our other sites that use Airbrake